src/test/pmem_map_file/pmem_map_file.c

```diff
@@ -184,11 +186,14 @@ main(int argc, char *argv[])
 		else
 			is_pmemp = NULL;
 
-		UT_OUT("%s %lld %s %o %d %d",
-			path, len, argv[i + 2], mode, use_mlen, use_is_pmem);
+		UT_OUT("%s %lld %s %o %d %d %d",
+		path, len, argv[i + 2], mode, use_mlen, use_is_pmem, err_code);
 
 		addr = pmem_map_file(path, len, flags, mode, mlenp, is_pmemp);
 		if (addr == NULL) {
+			if (err_code != 0) {
```

Comment:
I don't see any test that sets this value.

Reply:
err_code is expected error code given as test parameter,
if err_code is equal 0 I can't compare it to errno

Reply:
I know what it does and how. All tests pass 0 as err_code, so I wonder what's the purpose of such change.

Reply:
You're right. Should I remove err_code parameter?

Reply:
I think it's the right choice if you won't use it in this PR.

Reply:
Ok, done.

Reply:
I checked this pull request and realize that I didn't push some test cases with err_num different from 0. I fixed this and restored previous version. Also I realize that some test cases are covered by existing tests.